experimenter/experimenter/targeting/constants.py

```diff
@@ -2681,7 +2681,7 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
-TOU_NOT_ACCEPTED_ADS_ENABLED_MAC_OR_WIN = NimbusTargetingConfig(
+TOU_NOT_ACCEPTED_EXISTING_USER_ADS_ENABLED_MAC_OR_WIN = NimbusTargetingConfig(
     name="TOU not accepted yet, ads enabled, Mac or Win",
     slug="tou_not_accepted_ads_enabled_mac_win",
     description=(
```

Comment:
I like that you've updated the constant name so it still reflects what the constant does.  I'd suggest also updating the `name`, `slug`, and `description`. 

If we do this, we'll also need to make sure that the change of the `name` and `slug` don't confuse experimenter.

If we don't do it, it seems like it's a footgun that has some (somewhat small) chance of confusing us in (eg) debug messages, review, etc for this experiment. And a larger chance if for some reason we need to change or reuse the targeting in some other experiment along this whole path.

Either way, I think we would do well to have another round of QA.